src/components/picker/index.tsx

```diff
@@ -50,7 +49,9 @@ const DIALOG_PROPS = {
   height: 250
 };
 
-const Picker = (props: PropsWithChildren<PickerProps> & ForwardRefInjectedProps & BaseComponentInjectedProps) => {
+const Picker = (props: PropsWithChildren<PickerProps> & ForwardRefInjectedProps /*  & BaseComponentInjectedProps */
+) => {
```

Comment:
No need to break line here... Why leave the commented BaseComponentInjectedProps? If we don't use it, let's delete it

Reply:
Fixed